packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -7,6 +7,148 @@ section: get-started
 import './get-started.css';
 import { Divider } from '@patternfly/react-core';
 
+
+## PatternFly 5.4
+The final minor release within PatternFly 5 is now live! Next up is PatternFly 6. Don't forget to keep an eye on [our staging site](https://staging-v6.patternfly.org/) for updates. 
+
+But first, we have *lots* to share for this update&mdash;check out the highlights.
+
+### Promoted package versions
+
+- need list
+
+### Component updates
+#### Fixed bugs
+
+There were still a handful of bugs on our to-do list. To address a variety of reported issues, we did the following:
+
+- Added support for additional label info on horizontal forms.
+- Added a missing border on compound expansion table rows.
+- Fixed `<ExpandableSection>` errors that were caused by `variant="truncate"`
+- Fixed issues with icon variants in the "next" modal component.
+- Updated `paginationAriaLabel` to stop adding "items" to the end of labels.
+- Fixed unexpected wrapping in the pagination component. 
+- Added `shouldFocusFirstMenuItemOnOpen` to the dropdown component, to fix autofocus issues.
+- Fixed spacing issues with progress stepper's help text title.
+- Fixed some charts-related errors.
+- Added new `shouldFocusContent` property to the wizard component to help navigate accessibility challenges when navigating wizards with keyboard controls.
+- Improved accessibility features for the tree view component via aria properties.
+- Fixed an issue in the code block component that was removing code styling terms, like "\n" and "\t".
+- Updated the truncate component to only show a tooltip when truncation is present.
+- Drag and drop: 
+  - Switched to the flex layout in the multi-list drag and drop example.
+  - Fixed the positioning of `DragOverlay`
+
+#### New features and demos
+
+We added new features to a few of our components, along with new examples and demos, including:
+- Expanded selectable area size for checkboxes.
+- New page component properties to support drawer width customization.
+- Multiple drop zones in the "next" drag and drop component.
+- [Editable table rows.](/components/table#editable-rows)
+- [Main section variations for the page component.](/components/page#main-section-variations)
```

Comment:
Do you know what this is referring to? I looked at that example and it wasn't obvious what was added.

Reply:
Hm which part? The page stuff? It looks like this new example may be what I was referring to as far as "main section variations": https://github.com/patternfly/patternfly-react/pull/10352/files. And I think this one for drawer width: https://github.com/patternfly/patternfly-react/issues/10133